src/main/java/uk/co/ramp/covid/simulation/place/Hospital.java

```diff
@@ -81,11 +81,20 @@ public void reportDeath(DailyStats s) {
     }
 
     @Override
-    public double getTransP(Time t, Person infected, Person target) {
+    public double getTransP(Person infected) {
         double transP = getBaseTransP(infected);
         if (infected.isHospitalised()) {
             transP *= CovidParameters.get().hospitalisationParameters.hospitalisationTransmissionReduction;
         }
         return transP;
     }
+    
+    @Override
+    public double getEnvironmentAdjustment(Person susceptible, Person infected, Time t) {
+    	
+    	if(!susceptible.isHospitalised() && infected.isHospitalised() && infected.goesToHosptialInPhase2()) {
```

Comment:
I don't think we need the goesToHospital check since you can only ever get hospitalised if that's the case (hospitalised people are only COVID, never inPatients who are treated more like visitors).

Reply:
I did this in anticipation of other patients being hospitalised to differentiate those that were in a Covid hospital form those with Covid in a normal ward who transmit normally. See the adjusted please